Computation.hs

```diff
@@ -24,6 +26,11 @@ below. Or you can let the compiler do it with a SPECIALIZE pragma, see e.g. the
 one for msComputation+Fused below.
 -}
 
+heffComputation :: (HEffect.Carrier sig m, HEffect.Effect sig) => Int -> HEffect.Eff (HEffect.StateC Int m) ()
```

Comment:
Would be really curious to see if the benchmarks are significantly different if these are left abstracted over `m` completely, i.e.:

```haskell
heffComputation :: (HEffect.Carrier sig m, HEffect.Effect sig, Monad m) => Int -> m ()
```

Reply:
We could use a `SPECIALIZE` pragma, if necessary.